src/py/flwr/common/record/typeddict.py

```diff
@@ -73,3 +84,15 @@ def __eq__(self, other: object) -> bool:
         if isinstance(other, dict):
             return data == other
         return NotImplemented
+
+    def keys(self) -> KeysView[K]:
+        """D.keys() -> a set-like object providing a view on D's keys."""
```

Comment:
```suggestion
        """Provide a view on the dictionary's keys."""
```

Reply:
Hmmmm, my intention is to keep `TypedDict` as close to Python `dict` as possible, and thus I copied and pasted the docstring for `dict.keys`. But I am also happy to make the docstring clearer. Wdyt?

Reply:
oh, you are right! I guess i never paid much attention to the docstrings for `.key()` etc. Let's keep it as you had 💯